netbox/dcim/svg/cables.py

```diff
@@ -181,6 +182,8 @@ def _get_color(cls, instance):
         if hasattr(instance, 'role'):
             # Device
             return instance.role.color
+        elif instance._meta.model_name == 'circuit':
```

Comment:
Need to check whether `color` is actually set. (Device role, for instance, _requires_ a color, so this isn't a concern above.)